### DISCUSSION TITLE
IE 11: No events are fired. It seems to not bind anything at all

### DISCUSSION THREAD

**Discussion by iMusicJJ** [alpinejs/alpine#1398] [7. Bugs (archived)]
Hi,

Everything runs smoothly on Chrome, Firefox, Edge etc.

However, when I try to run the IE11 build, with something as simple as `x-init="alert('test')"` nothing happens. Opening the page in Chrome, Firefox etc. yields the expected alert popup.

Javascript is enabled, and I have this in the head:
```
<script type="module" src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.x.x/dist/alpine.min.js"></script>
<script nomodule src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.x.x/dist/alpine-ie11.min.js"></script>
```

I can't get any reactivty at all in IE11, am I missing a step somewhere?

**Comment by HugoDF** [comment]
What you've described should work.

I don't have access to IE11 at the moment. Could you share a snippet that's not working in IE11?

**Comment by iMusicJJ** [comment]
Hi Hugo,

I'll have a reproducible codepen for you sometime during the weekend.

**Comment by iMusicJJ** [comment]
It seems to somehow be our stack. I am still investigating. I get no errors, no conflicts, but it works flawlessly on Edge, Chrome, Firefox etc. but IE11 refuses to work
